internal/pkg/cli/interfaces.go

```diff
@@ -46,6 +46,10 @@ type cwlogService interface {
 	LogGroupExists(logGroupName string) (bool, error)
 }
 
+type addonsService interface {
```

Comment:
Thanks for the sharing. Really helpful! But I have a concern here: if the interface is defined at the file we use then I think `Templater` is definitely better. However, it is in a file with many interfaces. Using the name `Templater` might not be clear enough since there could be other package also has `Template()` function and could be a collision in future. What do you think?

Reply:
Yeah good point but I think it's safe to assume that all various implementations of `Template()` would return `(string, error)` so I think it's safe and even more generic!

Reply:
Oh yeah I agree